streams/src/test/java/org/apache/kafka/streams/processor/internals/AssignedTasksTest.java

```diff
@@ -220,7 +221,40 @@ public void shouldCloseTaskOnSuspendIfTaskMigratedException() {
         EasyMock.expectLastCall();
         EasyMock.replay(t1);
 
-        assertThat(suspendTask(), nullValue());
+        suspendTask();
+
+        assertTrue(assignedTasks.previousTaskIds().isEmpty());
+        EasyMock.verify(t1);
+    }
+
+    @Test
+    public void shouldCloseTaskOnCloseWhenRuntimeException() {
```

Comment:
Could we verify that `task.close(clean = false)` is called then?

Reply:
We set up this check via
```
t1.close(false, false);
EasyMock.expectLastCall();
```

and verify at the end: `EasyMock.verify(t1);`